src/current/v23.1/automatic-cpu-profiler.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Automatic CPU Profiler
+summary: The Automatic CPU Profiler captures CPU profiles automatically to help investigate cluster issues involving CPU load.
+keywords: cpu profile, cpu profiles, cpu profiler, automatic cpu profiler
+toc: true
+docs_area: manage
+---
+
+{% include_cached new-in.html version="v23.1" %} This feature automatically captures CPU profiles. A CPU profile shows the functions that use the most CPU time, sampled over a window of time.
+
+Use CPU Profiles to troubleshoot spikes in CPU utilization or erratic CPU load on certain nodes. You can collect a CPU Profile manually on the [Advanced Debug page](ui-debug-pages.html). However, it may be difficult to manually capture a profile of a short CPU spike at the right point in time. Automatic CPU profile capture enables the investigation of CPU load in this and other cases, such as periodic high CPU.
+
+{{site.data.alerts.callout_danger}}
+We strongly recommend only using the Automatic CPU Profiler when working directly with the [Cockroach Labs support team](support-resources.html).
+{{site.data.alerts.end}}
+
+## Configuration
+
+You can configure automatic CPU profile capture with the following [cluster settings](cluster-settings.html):
+
+Cluster Setting | Description | Default Value | Recommended Value
+----------------|-------------|---------------|------------------
+`server.cpu_profile.enabled` | The ON/OFF switch of the Automatic CPU Profiler | `false` | 
+`server.cpu_profile.cpu_usage_combined_threshold` | The baseline value for when CPU profiles should be taken. Collect profiles from each node that meets threshold. | MAX integer, such as `9223372036854775807` | `80`
+`server.cpu_profile.interval` | The period of time after which the high-water mark resets to the baseline value | `5m0s` (5 minutes) | `1m40s` (100 seconds)
```

Comment:
Should "high-water mark" link to the high-water mark section below?

Reply:
Added link to High-water mark threshold section.